web/tests/delete-test.spec.ts

```diff
@@ -0,0 +1,36 @@
+// Tests box deletion
+import { test, expect } from '@playwright/test';
+import { Workspace } from './lynxkite';
+
+let workspace: Workspace;
+test.beforeAll(async ({ browser }) => {
+  workspace = await Workspace.empty(browser);
+  await workspace.addBox({ id: 'ex0', name: 'Create example graph', x: 100, y: 100 });
+  await workspace.addBox({
+    id: 'pr1',
+    name: 'Compute PageRank',
+    x: 100,
+    y: 200,
+    after: 'ex0',
+    params: { name: 'page_rank_default', iterations: '1' },
+  });
+  await workspace.addBox({
+    id: 'pr2',
+    name: 'Compute PageRank',
+    x: 100,
+    y: 300,
+    after: 'pr1',
+    params: { name: 'page_rank_incoming', direction: 'incoming edges', iterations: '1' },
+  });
+});
+
+test('delete boxes', async () => {
+  await workspace.clickBox('pr2');// To check that only popups for deleted boxes are closed.
```

Comment:
Use `openBoxEditor()`. It has a clearer intention than `clickBox()`. It also returns the popup, so you can later use that in the assertion.

Also open the popup for `pr1`. Like the comment says, we want to see that the popups for deleted boxes are gone. But in your test the popup for the deleted box is not even open.